ports/stm32/uart.c

```diff
@@ -988,9 +1059,14 @@ STATIC const mp_rom_map_elem_t pyb_uart_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_readchar), MP_ROM_PTR(&pyb_uart_readchar_obj) },
     { MP_ROM_QSTR(MP_QSTR_sendbreak), MP_ROM_PTR(&pyb_uart_sendbreak_obj) },
 
+    // \method irq(handler=None, trigger=)
+    { MP_ROM_QSTR(MP_QSTR_irq), MP_ROM_PTR(&pyb_uart_irq_obj) },
+
     // class constants
     { MP_ROM_QSTR(MP_QSTR_RTS), MP_ROM_INT(UART_HWCONTROL_RTS) },
     { MP_ROM_QSTR(MP_QSTR_CTS), MP_ROM_INT(UART_HWCONTROL_CTS) },
+    // IRQ flags NO irq is defined in pyb_irq
+    { MP_ROM_QSTR(MP_QSTR_RX_IDLE_IRQ), MP_ROM_INT(IRQ_FLAG_RX_IDLE) },
```

Comment:
I think the constants should start with IRQ, following existing `Pin.IRQ_RISING`, `Pin.IRQ_FALLING`, so `UART.IRQ_RX_IDLE`.